setup.cfg

```diff
@@ -35,6 +35,10 @@ testing =
 [options.package_data]
 cdo_local_uuid = py.typed
 
+[options.packages.find]
+# https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#finding-simple-packages
+exclude = tests
```

Comment:
This `exclude` is to prevent the demonstration applications under `tests/relativity_to_venv` from being picked up as part of the package being sent to `pypi`.

I'll disclaim that I haven't used this directive before.  We can try using test PyPI if we aren't sure this exclude will work on the first try.